tests/e2e/dns-manipulation/create/README.md

```diff
@@ -0,0 +1,56 @@
+# Test: `netpol-adapter-policy-creation`
+
+This test validates that the netpol policy is being created when a user applies a intent and intentbinding.
```

Comment:
Why are we making changes to dns manipulation ?

Reply:
didn't get the point @shivaccuknox .